documentation/asciidoc/topics/proc_defining_environment_var.adoc

```diff
@@ -0,0 +1,51 @@
+[id='defining-environment-variables_{context}']
+= Defining environment variables in the {ispn_operator} subscription
+
+[role="_abstract"]
+Define environment variables in your {ispn_operator} subscription that automatically propagate to each {brandname} po and service managed by the corresponding Operator.
+
+`spec.config.env` field is an array of environment variable definitions.
+You can add one or more environment variable names and corresponding values.
```

Comment:
Do you have any recommendations here? What the environment variables should not be used for? Best practices etc? 
For example, Kubernetes docs mention " Environment variables may reference each other, however ordering is important. Variables making use of others defined in the same context must come later in the list. Similarly, avoid circular references." & "The environment variables set using the env or envFrom field override any environment variables specified in the container image."

Reply:
> Do you have any recommendations here?

I think the `TZ` environment variable in the Jira is a good one as we have users who actively need to change the timezone associated with their deployments.


> What the environment variables should not be used for? Best practices etc?

I think it's sufficient to just state that the order of the variables is important if one env var references another.